### PR TITLE
Delete check priority number to support newer version api

### PR DESCRIPTION
**Pull request by DENNKA**
priority numbers changed in v4.1+
0 | Do not download
1 | Normal priority
6 | High priority
7 | Maximal priority

### CODE REVIEW

**Comment by siriak** [comment]
Let's just update them to the new version, why remove the check?

**Comment by DENNKA** [comment]
> Let's just update them to the new version, why remove the check?

I thought package also works with older api, i changed priorities to 4.1+

**Comment by siriak** [comment]
@v1k45 could you take a look at this change?

File: qbittorrent/client.py [hunk 1]
```diff
@@ -615,9 +615,8 @@ def set_file_priority(self, infohash, file_id, priority):
         :param infohash: INFO HASH of torrent.
         :param file_id: ID of the file to set priority.
         :param priority: Priority level of the file.
-        :note priority 4 is no priority set
         """
-        if priority not in [0, 1, 2, 4, 7]:
+        if priority not in [0, 1, 6, 7]:
             raise ValueError("Invalid priority, refer WEB-UI docs for info.")
```
Comment (v1k45) [inline qbittorrent/client.py]: Should we keep 2, 4 for backwards compatibility?

Comment (siriak) [reply qbittorrent/client.py]: I'd say yes, it's easier than versioning the scripts

Comment (siriak) [reply qbittorrent/client.py]: Maybe with some comments as to which priorities are relevant to which qbittorrent version

Comment (DENNKA) [reply qbittorrent/client.py]: where from priority 4? In github docs wrote
4.1+ 0, 1, 6, 7
3.2.0-4.0.4 also 0, 1, 6, 7
3.1.x 0, 1, 2, 7

**Review by siriak — APPROVED** [review]

**Review by v1k45 — APPROVED** [review]
Thanks for your contribution. I'll push a release later this week.

**Pull request merged**